ui/src/app/pages/workspace/workspace-library.tsx

```diff
@@ -40,17 +39,40 @@ const styles = reactStyles({
   },
   menuLinkSelected: {
     backgroundColor: colorWithWhiteness(colors.primary, 0.95)
+  },
+  libraryTabDivider: {
+    width: '100%',
+    margin: '0.5rem 0',
+    backgroundColor: colorWithWhiteness(colors.dark, .5),
+    border: '0 none',
+    height: 1
   }
 });
 
 const libraryTabEnums = {
-  FEATURED_WORKSPACES: {
-    title: 'Featured Workspaces',
-    icon: 'star'
-  },
   PUBLISHED_WORKSPACES: {
     title: 'Published Workspaces',
-    icon: 'library'
+    icon: 'bookmark',
+    filter: (publishedWorkspaces: WorkspacePermissions[], featuredWorkspaces: FeaturedWorkspace[]) => {
+      return publishedWorkspaces.filter(ws => !featuredWorkspaces.find(fws =>
```

Comment:
Optional nit: I'm generally averse to overly-short abbreviations, because they can end up being difficult to grok and chars are cheap. But I'm happy to acknowledge this battle is lost if we have a strong pattern in our codebase to use 1-3 letter variable names. Do you have a clear sense on this?

Reply:
I'm all for longer variable names. The few times I break that rule is if I'm writing a small-ish inline function but I'm happy to change to their full names

Reply:
Yeah, +1 for tiny var names if they're going to be extremely short-lived (i.e. 1-line lambda or equivalent) and the value/context is obvious. Or in situations where we need to access it 10 times in a row in a conversion or factory method, such as when doing what ObjectMapper does.